actioncable/lib/action_cable/server/configuration.rb

```diff
@@ -21,6 +21,7 @@ def initialize
 
         @connection_class = -> { ActionCable::Connection::Base }
         @worker_pool_size = 4
+        @executor_pool_size = 10
```

Comment:
This configuration seems specific to threaded execution, does it make sense in general or should we consider server specific configuration?